model/converter/json/fixtures/es_01-actual.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
this should not be checked in, it's created by the unit test for debugging (when there is a mismatch)

Reply:
Yes, will remove it and add it to gitingore